src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AbstractSymbolDescriptionBuilder.cs

```diff
@@ -372,7 +414,13 @@ private static int GetPrecedingNewLineCount(SymbolDescriptionGroups group)
             }
 
             private IDictionary<SymbolDescriptionGroups, ImmutableArray<TaggedText>> BuildDescriptionSections()
-                => _groupMap.ToDictionary(kvp => kvp.Key, kvp => kvp.Value.ToTaggedText());
+            {
+                var result = new Dictionary<SymbolDescriptionGroups, ImmutableArray<TaggedText>>(_documentationMap);
+                foreach (var (group, parts) in _groupMap)
+                    result[group] = parts.ToTaggedText();
```

Comment:
I don't understand this. Why are we replacing documentation text with symbol text?

Reply:
there are two maps.  one that goes from group->symboparts, and one that goes from group->taggedtext. the maps cover different groups.  So this just produces a final merged dictionary that is entirely from group->taggedtext.